core/modules/filter/filter.admin.inc

```diff
@@ -14,7 +14,7 @@
  */
 function filter_admin_overview($form) {
   // Overview of all formats.
-  $formats = filter_formats();
+  $formats = filter_formats(NULL, TRUE, TRUE);
```

Comment:
The last `TRUE` can be removed.

Reply:
yeah, this was the one that came with the patch I made from your PR. Sorry about that, caught it and still forgot to delete...  killed now.